docs/mine/lotus/dagstore.md

```diff
@@ -0,0 +1,191 @@
+---
+title: 'Lotus Miner: About the markets dagstore'
+description: ''
+breadcrumb: 'About the markets dagstore'
+---
+
+# {{ $frontmatter.title }}
+
+{{ $frontmatter.description }}
+
+[[TOC]]
+
+## Conceptual overview
+
+::: tip
+The dagstore was introduced in Lotus v1.11.2.
+:::
+
+The dagstore is a sharded store to hold large IPLD graphs efficiently, packaged as location-transparent attachable CAR files, with mechanical sympathy resulting in zero-copy access in ideal situations.
+
+The dagstore is a component of the markets subsystem in `lotus-miner`. It replaces the former Badger staging blockstore. It is designed to provide high efficiency and throughput, and minimize resource utilization during deal-making operations.
+
+The dagstore leverages the indexing features of [CARv2](https://github.com/ipld/ipld/blob/master/specs/transport/car/carv2/index.md) to enable plain CAR files to act as read and write blockstores. These CAR files serve as the direct medium for data exchange in markets and deal-making processes, without requiring intermediate buffers.
+
+Compared to before v1.11.2:
+
+* On storage deals, miners no longer stage the data into a Badger store prior to forming the unsealed CAR. Instead, the unsealed CAR is built directly as the data is being transferred through graphsync.
+* On retrieval deals, miners no longer load the unsealed CAR into a Badger blockstore first. Retrievals are served directly from the data at rest (the unsealed CAR).
+
+The DAG store comprises three layers:
+
+1. Storage layer (manages shards).
+2. Index repository (manages indices).
+3. DAG access layer (manages queries).
+
+## Terminology
+
+Here's a definition of terms:
+
+- **Shard:** the unit of data storage in the dagstore. Every Filecoin piece (the data within a deal) is a unique shard in the dagstore. Every shard is identified by a unique **key**, whose value is the **Piece CID**.
+- **Shard state:** the state that a dagstore shard is in. It can be one of the  following:
+  - `ShardStateNew`: indicates that a shard has been registered, but has not yet been initialized.
+  - `ShardStateInitializing`: indicates that the shard is being initialized.
+    - `ShardStateAvailable`: indicates that the shard has been initialized and is capable of serving retrievals. However, there are no active shard readers in this instant.
+  - `ShardStateServing`: indicates the shard has active readers, and thus is currently serving at least one retrieval.
+    - `ShardStateErrored`: indicates that an unexpected error was encountered during a shard operation (e.g. initialization or acquisition), and therefore the shard needs to be recovered to be fully operational.
```

Comment:
What’s shard acquisition and what does it do?

Reply:
Explained in later lines - maybe link?

Reply:
Just add a comment "explained below".